packages/rspack-test-tools/tests/__snapshots__/NewCodeSplitting-stats-output.test.js.snap

```diff
@@ -64,8 +64,7 @@ Rspack compiled successfully (27eddb11e03f28d2)
 exports[`new code splitting stats output new code splitting stats output/builtin-swc-loader-parse-error should print correct stats for: NewCodeSplittingStatsOutput 1`] = `
 ERROR in ./index.ts
   × Module build failed:
-  ├─▶   ×
-  │     │   x Expected '{', got 'error'
+  ├─▶   ×   x Expected '{', got 'error'
```

Comment:
There are two `× x` in one line, can we dedupe them?

Reply:
Two errors, nested one is formatted by swc, the outer one is formatted by us.

Reply:
ok, seems hard to handle